src/main/starlark/builtins_bzl/common/cc/cc_helper.bzl

```diff
@@ -532,6 +532,9 @@ def _is_versioned_shared_library_extension_valid(shared_library_name):
             return True
     return False
 
+def _is_versioned_shared_library_extension_valid(shared_library_name):
```

Comment:
Instead of adding this, could we rename _is_versioned_shared_library_extension_valid to _is_versioned_library_extension_valid and do a minor refactoring in cc_helper.bzl? _is_versioned_shared_library_extension_valid is private and not exported so it should just touch one file.